trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.delete(' ')
+  end
```

Comment:
- [ ] As the previous reviewer suggested, Kindly work on the trimmer method. :+1: 

> **Project requirement:** Implement a method correct_name that makes sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.